### PR TITLE
new autoloader, draft working api using namespaces

### PR DESCRIPTION
**Pull request by will123195**

### CODE REVIEW

File: lib/class/class.aql.php [hunk 1]
```diff
@@ -1,898 +1,898 @@
...
+						objects => optional
+	@params		(array) generated by aql2array
+				(array) clause array
+
+**/
+
+	public function make_sql_array($arr, $clause_array = null) {
+		
+		if (count($arr) == 0) {
+			throw new Exceptino('AQL Error: You have an error in your syntax.');
```
Comment (stanistan) [inline lib/class/class.aql.php]: Typeo found.

File: lib/Sky/Autoloader.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?
```
Comment (stanistan) [inline lib/Sky/Autoloader.php:1]: Inconsistent indentation in this file (spaces/tabs)

**Comment by stanistan** [comment]
We need to have standard documentation on each of these classes. (DocBloc)

Is `/Sky/Api/Resource` not supposed to be used on it's own? The constructor doesn't do anything with the arguments, nothing gets set. If it isn't supposed to be used on it's own (must be extended), it should probably be declared abstract, and the `__construct` method as abstract as well.

File: lib/Sky/Api.php [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+			}
+		}
+		$this->output->meta->status = 'ok';
+		return $this->output;
+	}
+
+
+	/**
+	 * return the error message in a standardized format
+	 * @return \Sky\Api\Response
```
Comment (stanistan) [inline lib/Sky/Api.php:221]: @param string $message

File: lib/Sky/Autoloader.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+			include $filename;
+			if (class_exists($className)) return true;
+		}
+		return false;
+	}
+
+	/**
+	 *	loads class files into the global space
+	 *		lib/class/class.{className}.php
+	 *	@return boolean
```
Comment (stanistan) [inline lib/Sky/Autoloader.php]: @param

**Comment by will123195** [comment]
All fixes have been made.  Event class still needs documentation.

**Comment by will123195** [comment]
fixed @params

**Pull request merged**